src/frontend/src/pages/GanttPage/GanttPackage/helpers/bar-helper.ts

```diff
@@ -33,12 +29,12 @@ export const convertToBarTasks = (
       barCornerRadius,
       handleWidth,
       rtl,
-      barProgressColor,
-      barProgressSelectedColor,
+      t.styles?.backgroundColor ?? '#9d9d9d',
```

Comment:
i dont think this is a great place for the defaults to be since no other defaults are in this file

Reply:
this is the only place we map individual styles for tasks

Reply:
make the default be the barProgressColor